fundingmanager.go

```diff
@@ -196,15 +196,15 @@ type fundingConfig struct {
 	// announcement from the backing Lightning Network node.
 	CurrentNodeAnnouncement func() (lnwire.NodeAnnouncement, error)
 
-	// SendAnnouncement is used by the FundingManager to send announcement
+	// SendAnnouncement is used by the Manager to send announcement
 	// messages to the Gossiper to possibly broadcast to the greater
 	// network. A set of optional message fields can be provided to populate
 	// any information within the graph that is not included in the gossip
 	// message.
 	SendAnnouncement func(msg lnwire.Message,
 		optionalFields ...discovery.OptionalMsgField) chan error
 
-	// NotifyWhenOnline allows the FundingManager to register with a
+	// NotifyWhenOnline allows the Manager to register with a
```

Comment:
I don't really think we need to change these comments. talking about the `FundingManager` is okay even though it is named `Manager` now

Reply:
yeah